src/main/scala/com/databricks/spark/util/DatabricksLogging.scala

```diff
@@ -31,7 +31,7 @@ object TagDefinitions {
   object TAG_OP_TYPE extends TagDefinition
 }
 
-class OpType(typeName: String, description: String)
+class OpType(val typeName: String, val description: String)
```

Comment:
@zsxwing , do you know why this isn't just a case class?

Reply:
Want me to change the PR to be a case class?

Reply:
This is probably a mistake when we open-sourced the codes. This should be a case class. @AFFogarty Yep, just go ahead and make it a case class.

Reply:
Changed to case class